README.md

```diff
@@ -10,7 +10,46 @@ First, you'll need to configure a database. Once this project is deployed, we'll
 
 ### Windows setup
 
-Coming Soon!
+## Chocolatey and Postgres: (Needs Testing)
+1. [Install Chocolatey](https://chocolatey.org/docs/installation#installing-chocolatey) if you do not already have it installed.
+2. Add `C:\ProgramData\chocolatey\bin` to your paths (type path in search, click Edit environment varaibles for you account).
+3. In your powershell (admin), run `choco install postgresql10` (powershell (admin) can be found by pressing `win + x`). 
+- Make sure you use the admin powershell throughout all the steps.
+4. Upgrade postgresql: `choco upgrade postgresql`.
+5. Add `C:\Program Files\PostgreSQL\10\bin` to your [paths](https://helpdeskgeek.com/windows-10/add-windows-path-environment-variable/).
+6. Restart your admin terminal to update the path. 
+7. Start postgres: `psql -U postgres` you will see postgres-# in the terminal.
+8. Create your user with `CREATE USER name;` The terminal will print CREATE ROLE.
+9. Alter your role with permission to create a database with `ALTER USER name WITH CREATEDB CREATEROLE;` The terminal will print ALTER ROLE
+10. Check everything is in order with `\du`. The terminal will print a table with Role name, Atrributes, and Member of columns. If your user is listed you are good to go.
+11. Quit with `\q` or `ctr + c`.
+12. Start psql with new username with `psql`. You should see `name=>`.
+13. Create a database with: `CREATE DATABASE resources OWNER name;`.
+14. Connect to the resources database: `\c resources`. If you see resources => you are ready to move on to the next steps.
+## Set Up and Activate your Environment:
+15. Set the connection information in a [user environment variable](https://docs.microsoft.com/en-us/windows/desktop/shell/user-environment-variables):
```

Comment:
I think we can add this step to "Step 2" and add all of the variables all in one place, huh?

Reply:
fixed